components/Layout/types.ts

```diff
@@ -1,6 +1,7 @@
 type LayoutProps = {
     title: string;
     lastPublishedAt: string;
+    meta: { [key: string]: string | boolean | null };
```

Comment:
**Nitpick**: This is a neat type definition shorthand, but as we don't need to support completely custom keys in our meta object (just `socialText`, `searchDescription`, `socialImageUrl`, `seoTitle`, `slug`) I'd prefer if these fields were specifically defined in the `meta` props type definition. This way you also don't have to recast the object field types every time you use them in `Layout.tsx`

Reply:
Nice suggestion. I've updated it. Thanks!